src/components/Finance.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+import Input from './Input';
+
+import './dashboard.css';
+
+const Finance = () => {
+
+    return (
+        <>
+            <h1>BUDGET DASHBOARD</h1>
```

Comment:
can the header element not go inside the section?

Reply:
This header is indicating this is a dashboard, could you not change the name of the component to reflect that?